yappi/_yappi.c

```diff
@@ -1314,7 +1432,7 @@ _start(void)
     }
 
     if (flags.multithreaded) {
-        _enum_threads(&_profile_thread);
+        _enum_threads(&_bootstrap_thread);

```

Comment:
How about following?

```
if CTX_BACKEND == THREAD
if (flags.multithreaded) {
        _enum_threads(&_profile_thread);
    } else {
        _ensure_thread_profiled(PyThreadState_GET());
        initial_ctx = _thread2ctx(PyThreadState_GET());
    }
ELIF CTX_BACKEND== GREENLET
do whatever
```

I think it would better to be explicit in the code for ctx_backend.

Reply:
I have replaced `multithread` with `multicontext` in yappi.c. This was simpler than using two separate variables - `multigreenlet` and `multithread` considering that almost everything that must be done for multithread is also done for multigreenlet.

In the context of this piece of code, `_enum_threads(&profile_thread)` needs to be called for greenlets as well because there are systems with multiple native threads running multiple greenlets on each. The system at our company is like this.